src/slscommands/moderation/clear.js

```diff
@@ -0,0 +1,45 @@
+module.exports = {
+  name: 'clear',
+  description: 'To delete some messages in channels.',
+  permissions: ['MANAGE_MESSAGES'],
+  type: 1,
+  options: [
+    {
+      name: 'number',
+      description: 'Give the number of message deleted.',
+      required: true,
+      type: 'NUMBER',
```

Comment:
You pass a `min_value` & `max_value`  :
![image](https://user-images.githubusercontent.com/31402774/172050070-091252b1-d4e0-4db6-8581-995357bc5087.png)
```suggestion
      type: 'NUMBER',
      min_value: 1,
      max_value: 99
```